examples/vue/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@aws-amplify/ui-vue": "^2.1.0",
-    "aws-amplify": "^4.1.3",
+    "aws-amplify": "latest",
```

Comment:
Should we do the same for other packages in the repo? They are in `^4.3.13`

Reply:
+1

Reply:
Yes, I will update this PR to include all packages.